src/handlers/net/NetStreamHandler.h

```diff
@@ -117,7 +121,7 @@ class NetworkMetricsBucket final : public visor::AbstractMetricsBucket
         , _throughput_out("payload", {"rates", "bytes_out"}, "Rate of egress throughput in bytes per second")
     {
         set_event_rate_info("packets", {"rates", "pps_total"}, "Rate of all packets (combined ingress and egress) in packets per second");
```

Comment:
@rboucher-ns1  should we rename rates metrics as well?

Because we rave rates pps_in pps_out and this total which does not consider filtered data

Reply:
Are you saying that `pps_in` and `pps_out` metrics are calculated for for ALL packets, not just the filtered packets?

Reply:
no I'm saying that `pps_in` and `pps_out` is calculated for packets not filtered out. but `pps_total` is calculated for ALL packts, 
so for instance:
 if we have like 100 packets per second and all being **filtered out**
 
 `packets_in` counter will be 0
 `pps_in` rates will be 0
 `packets_out` counter will be 0
 `pps_out` rates will be 0
 
 `packets_total` will be 0
 packets_filtered will be like 6000 (60*100)
 packets_events will be like 6000 (60*100)
 
 and `pps_total` rates will be 100


DO you think it confusing or not?

Reply:
Yes, it's confusing. `pps_total` should provide numbers that are aligned with `packets_total` (i.e. based on the same underlying data/packet set). 

The current `pps_total` should be renamed `pps_events` (or removed completely). 

A new `pps_total` (and a new `pps_unknown_dir`?) should be created to reflect _post_-filter packet rates

Reply:
`pps_unknown_dir ` and throughtput don't know if we really need that, we could get that from other metrics, what do you think?

Just updated the PR